### PR TITLE
fix wrong 'hasScroll' result for some IE specific case (part of #2511)

### PR DESCRIPTION
**Pull request by miherlosev**
@LavrovArtem @AndreyBelym @AlexKamaev

### CODE REVIEW

**Comment by miherlosev** [comment]
@testcafe-build-bot retest

**Comment by miherlosev** [comment]
@testcafe-build-bot retest

**Comment by miherlosev** [comment]
@testcafe-build-bot retest

**Comment by miherlosev** [comment]
FPR

File: src/client/core/utils/style.js [hunk 1]
```diff
@@ -24,7 +25,27 @@ export var setScrollLeft        = hammerhead.utils.style.setScrollLeft;
...
+    const overflowX            = get(el, 'overflowX');
+    const overflowY            = get(el, 'overflowY');
+    let scrollableHorizontally = SCROLLABLE_OVERFLOW_STYLE_RE.test(overflowX);
+    let scrollableVertically   = SCROLLABLE_OVERFLOW_STYLE_RE.test(overflowY);
+
+    // IE11 and MS Edge bug: There are two properties: overflow-x and overflow-y.
+    // If one property is set so that the browser may show scrollbars (`auto` or `scroll`) and the second one is set to 'visible',
+    // then the second one will work as if it had the 'auto' value.
+    if (browserUtils.isIE) {
+        if (!scrollableHorizontally && scrollableVertically && overflowX === DEFAULT_IE_SCROLLABLE_OVERFLOW_STYLE_VALUE)
```
Comment (churkin) [inline src/client/core/utils/style.js]:
```javascript
if (browserUtils.isIE) {
    scrollableHorizontally = scrollableHorizontally || scrollableVertically && overflowX === DEFAULT_IE_SCROLLABLE_OVERFLOW_STYLE_VALUE;
    scrollableVertically   = scrollableVertically || scrollableHorizontally && overflowY === DEFAULT_IE_SCROLLABLE_OVERFLOW_STYLE_VALUE;
}
```

File: src/client/core/utils/style.js [hunk 2]
```diff
@@ -129,8 +147,8 @@ export function hasScroll (el) {
     if (!scrollableHorizontally && !scrollableVertically)
         return false;
 
-    var hasHorizontalScroll = scrollableVertically && el.scrollHeight > el.clientHeight;
-    var hasVerticalScroll   = scrollableHorizontally && el.scrollWidth > el.clientWidth;
+    var hasVerticalScroll   = scrollableVertically && el.scrollHeight > el.clientHeight;
```
Comment (churkin) [inline src/client/core/utils/style.js]: Use `const`

**Comment by miherlosev** [comment]
@testcafe-build-bot retest

**Comment by miherlosev** [comment]
@LavrovArtem FPR

**Pull request merged**